Gemfile

```diff
@@ -5,22 +5,20 @@ ruby File.read(".ruby-version").strip
 gem "rails", "5.2.3"
 
 gem "decent_exposure", "~> 3.0"
+gem "gds-api-adapters", "~> 60.1"
+gem "govuk_app_config", "~> 2.0"
+gem "govuk_publishing_components", "~> 21.0.0"
 gem "jwt", "~> 2.2"
+gem "plek", "~> 3.0"
```

Comment:
Sorry to nitpick again, but the only change in this file appears to be the components gem version - everything else is lines moved around but remaining the same. If there's no reason for this, can you reorder the lines back to how they were so that the change is clear?

Reply:
the reason for this is the linting is insisting on alphabetical order since it changed in master prior to me pulling this branch. Believe me, I'm enjoying this far less than you...

Reply:
Ugh. Okay, I feel your pain. Ignore me then.